scripts/instaclustr-terraform-registry-documentation-update.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+INSTACLUSTR_API_URL="$IC_API_URL"
+
+if [ -z "$INSTACLUSTR_API_URL" ]
+then
+  INSTACLUSTR_API_URL="https://api.instaclustr.com"
```

Comment:
Isn't localhost going to be the more used default?

Reply:
Probably, yeah. But for production safety, the default is the production URL. In any case though, there's a simple make command to use to generate the docs for local testing - `make local-gen-docs`